atomicapp/run.py

```diff
@@ -55,6 +52,7 @@ def __init__(
             answers_format=ANSWERS_FILE_SAMPLE_FORMAT, **kwargs):
 
         self.debug = debug
+        self.display = Display()
```

Comment:
Why is `display` global var in one case and class attribute in another? I'd rather like to see this unified.